src/app/components/forms/post/_video/video.ts

```diff
@@ -196,16 +203,6 @@ export default class AppFormPostVideo extends BaseForm<FormModel>
 		this.maxAspect = $payload.maxAspect;
 		this.minAspect = $payload.minAspect;
 		this.allowedFiletypes = $payload.allowedFiletypes;
-
-		// If backend sends progress info, it means the attached uploaded video
-		// is being processed. However, if the user previously quit out of the
-		// video attaching, their previous video might still be processing in
-		// the background. We want to allow them to upload a new video instead.
-		// If they close and reopen the form without saving the post, their
-		// video from before will reappear.
-		if (this.canContinueProcessing && $payload.progress) {
```

Comment:
Since it's not needed anymore, remove the canContinueProcessing from both this form and the parent form.